pkgs/development/libraries/boehm-gc/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
   doCheck = true;
 
   # Don't run the native `strip' when cross-compiling.
-  dontStrip = stdenv ? cross;
+  dontStrip = crossSystem != null;
```

Comment:
That would be `hostPlatform`, existing names keep their old semantics. See my "confession" on names above.